### PR TITLE
Zjr/site migration

### PR DESCRIPTION
**Pull request by Zach-Bob**
Create basic migration and basic schema for the site table. This moves us towards our goal of allowing a user to add and remove sites from the system.
A dependency problem is also fixed - as of version 3.0, ecto requires the ecto_sql dependency for `Ecto.Adapters.Postgres` so that was added to mix.exs

### CODE REVIEW

File: lib/situation_room/sites.ex [hunk 1]
```diff
@@ -0,0 +1,8 @@
+defmodule Situation_Room.Sites do
```
Comment (cpjolicoeur) [inline lib/situation_room/sites.ex:1]: In Elixir, we typically name the models for schema objects in the Singluar.  So this should be `SituationRoom.Site`

File: lib/situation_room/sites.ex [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  use Ecto.Schema
+
+  schema "site" do
```
Comment (cpjolicoeur) [inline lib/situation_room/sites.ex:4]: See note on previous commit about table names in the plural

File: mix.exs [hunk 3]
```diff
@@ -22,6 +22,7 @@ defmodule SituationRoom.MixProject do
   defp deps do
     [
       {:plug, "~> 1.13.5"},
+      {:ecto, "~> 3.7.2"},
```
Comment (cpjolicoeur) [inline mix.exs:25]: Squash this commit back into the first commit since that is where this line was originally removed

**Review by cpjolicoeur — CHANGES_REQUESTED** [review]

File: lib/situation_room/sites.ex [hunk 4]
```diff
@@ -0,0 +1,8 @@
+defmodule SituationRoom.Site do
```
[the remark this answers is not in the document]
Comment (Zach-Bob) [reply lib/situation_room/sites.ex:1]: I have that commit in PR 33. Should I cherry-pick that commit and bring it in this PR or is it fine to leave it in the next PR? As it stands currently, PR 33 should be ready to merge right after this one.

**Comment by Zach-Bob** [comment]
This PR will be encompassed into the next PR

**Pull request closed without merging**